qiskit_ibm/ibm_provider.py

```diff
@@ -123,53 +114,36 @@ class IBMProvider(Provider):
 
     Note:
         The ``backend`` attribute can be used to autocomplete the names of
-        backends available to this provider. To autocomplete, press ``tab``
+        backends available to this account. To autocomplete, press ``tab``
         after ``provider.backend.``. This feature may not be available
         if an error occurs during backend discovery. Also note that
         this feature is only available in interactive sessions, such as
         in Jupyter Notebook and the Python interpreter.
     """
 
-    _credentials: Credentials = None
-    """Contains credentials of a new IBMProvider being initialized. If None, all the
-    provider instances have already been initialized and __new__ should return an
-    existing instance."""
-
-    _providers: Dict[HubGroupProject, 'IBMProvider'] = OrderedDict()
-
-    def __new__(
-            cls,
+    def __init__(
+            self,
             token: Optional[str] = None,
             url: Optional[str] = None,
-            hub: Optional[str] = None,
-            group: Optional[str] = None,
-            project: Optional[str] = None,
             **kwargs: Any
-    ) -> 'IBMProvider':
+    ) -> None:
         """IBMProvider constructor
 
         Args:
             token: IBM Quantum token.
             url: URL for the IBM Quantum authentication server.
-            hub: Name of the hub to use.
-            group: Name of the group to use.
-            project: Name of the project to use.
             **kwargs: Additional settings for the connection:
 
                 * proxies (dict): proxy configuration.
                 * verify (bool): verify the server's TLS certificate.
 
         Returns:
-            If `hub`, `group`, and `project` are specified, the corresponding provider
-            is returned. Otherwise the default provider is looked up in the
-            following order and returned:
-
-                * environment variables (QISKIT_IBM_HUB, QISKIT_IBM_GROUP and QISKIT_IBM_PROJECT)
-                * default_provider (hub/group/project) saved to disk
-                * open access provider (ibmq/open/main)
+            An instance of IBMProvider with services like IBMBackendService,
+                IBMRuntimeService, IBMExperimentService and IBMRandomService
+                as available to the account.
```

Comment:
I don't think it's necessary to list all the services, but if you want to, can you also add the links to them?